### ISSUE TITLE
Submission - Ignis Fatuus

### ISSUE THREAD

**Issue by nikpundik** [Algorithm-Arena/weekly-challenge-12-fools-cursor#4]
Inspired by a level from the Goblins game series, **Ignis Fatuus** is a super-minimal point-and-click game where the cursor **follows** the interest points.

## Video
[ignis.webm](https://github.com/Algorithm-Arena/weekly-challenge-12-fools-cursor/assets/4772224/15c945d2-956b-4998-9161-ad3a3e783931)

## Play
https://ignis-fatuus.vercel.app/

## Source
https://github.com/nikpundik/ignis-fatuus

**Comment by vjeux** [comment]
I gotta admit, I've been very confused by the game. I ended up winning but I didn't really understand the logic behind the game.

<img width="258" alt="image" src="https://github.com/Algorithm-Arena/weekly-challenge-12-fools-cursor/assets/197597/9c80756f-89f6-464a-a2e4-7a5c600ae1e9">

**Comment by nikpundik** [comment]
Yes this was probably a weak idea and rushed out..I was just trying to find a way to justify cursor orientation to help the finding of hidden buttons..the idea was to find hidden pieces of the solution by combining primary color switches on/off, white/black will reset it until you have them all and you can turn off black again. Definitely not the best design 😓 But i had cursor following the closer button so submitted anyway 🤕

**Comment by vjeux** [comment]
All good! The concept of finding the black one was really interesting, I just didn’t get how changing the colors connected to the password, and getting the last one by turning off black was a surprise.

This challenge is all about creativity and that definitely was <3
